lua/entities/gmod_wire_ledtape.lua

```diff
@@ -0,0 +1,340 @@
+AddCSLuaFile()
+DEFINE_BASECLASS( "base_wire_entity" )
+ENT.PrintName		= "Wire LED Tape Controller"
+ENT.WireDebugName 	= "LED Tape"
+
+function ENT:SharedInit()
+	self.Color = Color(255,255,255)
+	self.Path  = {}
+end
+
+Wire_LEDTape = Wire_LEDTape or {}
+
+Wire_LEDTape.MaxPoints = 256
```

Comment:
This should be 255. The NumBits on the following line will evaluate to 8 but 256 requires 9 bits.